ddtrace/contrib/trace_utils_llm.py

```diff
@@ -0,0 +1,135 @@
+import abc
```

Comment:
Can we make this module private? (ex: prefix it with an underscore). This will reduce the size of our public apii